logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -229,12 +235,25 @@ internal class MessageSenderImpl internal constructor(
         messageRepository.sendEnvelope(message.conversationId, envelope, messageTarget).fold({
             when (it) {
                 is ProteusSendMessageFailure -> messageSendFailureHandler.handleClientsHaveChangedFailure(it).flatMap {
-                    attemptToSend(message, messageTarget)
+                    attemptToSendWithProteus(message, messageTarget)
```

Comment:
Note: Changed this line, as at this point we know for sure the protocol, saving an extra query.